providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -180,6 +180,7 @@ class KubernetesPodOperator(BaseOperator):
         If more than one secret is required, provide a
         comma separated list: secret_a,secret_b
     :param service_account_name: Name of the service account
+    :param automount_service_account_token: If False, the pod will opt out of service account token automounting.
```

Comment:
```suggestion
    :param automount_service_account_token: indicates whether pods running as this service account should have an API token automatically mounted
```

To fix spell check. I used the doc string of the K8S SDK
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1ServiceAccount.md